project/core/settings/base.py

```diff
@@ -28,6 +28,7 @@
     'chapters',
     'home',
     'search',
+    'invites',
```

Comment:
Let's keep the custom apps in `INSTALLED_APPS` in alphabetical order.

```suggestion
    'home',
    'invites',
    'search',
```